completion/latex-document.cwl

```diff
@@ -14,9 +14,7 @@
 \appendixname#*
 \arabic{counter}#*
 \author{names}
-%<%:TEXSTUDIO-GENERIC-ENVIRONMENT-TEMPLATE%>
```

Comment:
this line generates the general "\begin{*environment*}"
The auto test fails because this line missing.
Please don't remove stuff just because it is not clear what it is doing ...

Reply:
Reverted. Sorry.